_src/scss/_settings.scss

```diff
@@ -161,7 +164,7 @@ $block-grid-max: 8;
 // 4. Base Typography
 // ------------------
 
-$header-font-family: Metamorphous, -apple-system, BlinkMacSystemFont, 'Segoe UI', system-ui, Roboto, 'Helvetica Neue', Arial, sans-serif, Apple Color Emoji, 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
+$header-font-family: $body-font-family;
```

Comment:
We only want to use Metamorphous for the largest headings (h1, h2), so I’ve separately set those to use the correct font in `_global.scss`.